src/Featurizers/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+# ----------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License
+# ----------------------------------------------------------------------
+cmake_minimum_required(VERSION 3.5.0)
+
+macro(Impl)
+    # Defining a function here to introduce a new scope for local variables
+    get_filename_component(_this_path ${CMAKE_CURRENT_LIST_FILE} DIRECTORY)
+
+    include(${_this_path}/cmake/FeaturizersCode.cmake)
```

Comment:
On line 9 you have `_this_path`, but on line 11 you have `${_this_path}`, was that intended?

Reply:
cmake syntax is awful - the usage is correct.

Reply:
Let's make it readable

---
In reply to: [356252729](https://github.com/microsoft/FeaturizersLibrary/pull/25#discussion_r356252729) [](ancestors = 356252729)